web-ui/src/api/team.js

```diff
@@ -1,39 +1,29 @@
 import { resolve } from "./api.js";
 
-const teamUrl = "/services/team";
-const teamMemberUrl = "/services/team/member";
+const teamUrl = `/services/team`;
 
-export const getAllTeamMembers = async (cookie) => {
-  return resolve({
-    method: "get",
-    url: teamMemberUrl,
-    responseType: "json",
-    headers: { "X-CSRF-Header": cookie },
-  });
-};
-
-export const getMembersByTeam = async (id, cookie) => {
-  return resolve({
-    method: "get",
```

Comment:
When you have a merge conflict on these, just leave them for now and I can do a refactoring story later